tests/test_files/calculator/calculator.py

```diff
@@ -1,5 +1,47 @@
-# TODO: Implement a calculator program, that can add, subtract, multiply and divide two numbers.
-#  - Numbers and operators given from the user as the command line arguments.
-#  - Consider using argpasre library.
-#  - Consider using eval builtin function.
-#  - Print or log the result.
+"""Calculator runnable
+This is a calculator program, that can add, subtract, multiply and divide two numbers. Numbers and operators given from the user as the command line arguments.
+The result is printed to the console.
+
+Args:
```

Comment:
Thit "Args:  ..." documentation can be deleted. Argparse shows it for the user and the developer as well